docs/data/base/components/no-ssr/no-ssr.md

```diff
@@ -7,29 +7,29 @@ packageName: '@mui/base'
 
 # No SSR
 
-<p class="description">NoSsr purposely removes components from the subject of Server Side Rendering (SSR).</p>
+<p class="description">The <code>NoSsr</code> component defers the rendering of children components from the server to the client.</p>
 
 This component can be useful in a variety of situations:
 
-- Escape hatch for broken dependencies not supporting SSR.
-- Improve the time-to-first paint on the client by only rendering above the fold.
-- Reduce the rendering time on the server.
-- Under too heavy server load, you can turn on service degradation.
-- Improve the time-to-interactive by only rendering what's important (with the `defer` property).
+- To create an escape hatch for broken dependencies that don't support server-side rendering (SSR)
+- To improve the time to first paint by only rendering above the fold
+- To reduce the rendering time on the server
+- To turn on service degradation when the server load is too heavy
+- To improve the Time to Interactive (TTI) by only rendering what's important (using the `defer` prop)
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-## Client-side deferring
+## Basic usage
+
+At its core, the `NoSsr` component's purpose is to **defer rendering** from the server to the client, as shown in the following demo:
 
 {{"demo": "SimpleNoSsr.js"}}
 
 ## Frame deferring
 
-At its core, the NoSsr component's purpose is to **defer rendering**.
-As it's illustrated in the previous demo, you can use it to defer the rendering from the server to the client.
-
-But you can also use it to defer the rendering within the client itself.
+You can also use `NoSsr` to defer rendering within the client itself.
 You can **wait a screen frame** with the `defer` property to render the children.
```

Comment:
What exactly is meant by "wait a screen frame"? Is there another way we can phrase this?

Reply:
Thank you @michaldudak !

Reply:
I rewrote this section for clarity, let me know what you think!

Reply:
I like it!

BTW - do you find the accompanying demo easy to understand? I have trouble understanding what's going on until I dive deeper into the code. The vertical size of the demo also doesn't make the best impression IMO.

Reply:
Actually now that you mention it, I really don't understand what that demo is showing. 😅 I thought I understood it at first glance but reviewing it again I don't think I get it.

Reply:
I'll go ahead and merge this revision, and we can come back to the demo in a separate PR.